docs/tutorial_bracket_strategy.md

```diff
@@ -174,50 +178,50 @@ Now, you should just follow the link and sign & execute the transaction with the
 For executing, just press the button _confirm_.
 
 If you are executing the second transaction with a lower gasPrice than the first one, then you can sign it immediately.
-Otherwise, you should wait until the first one is mined. This works as the gas price usually determines the mining order of the transaction.
+Otherwise, you should wait until the first one is mined. This works in most cases, as the gas price usually determines the mining order of the transaction.
 
-Congrats. You managed to provide liquidity and have a chance to earn a passive income, if the prices are moving and returning back to your initial currentPrice.
+Congrats! You just provided liquidity and have a chance to earn some passive income if the prices are moving and returning back to your initial `currentPrice`.
 
 ## Withdrawing liquidity
 
-The withdrawal from the Gnosis Protocol is always done in two step: requesting withdrawals and the actual withdrawal transferring the funds.
+The withdrawal from the Gnosis Protocol is always done in two steps: Requesting and Claiming.
 
 ### Withdraw request
 
 Withdrawal requests can be made with the following command:
 
 ```ssh
-npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma separated brackets]  --tokenIds=[indices] --requestWithdraw --network=$MASTER_SAFE
+npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --requestWithdraw --network=$MASTER_SAFE
 ```
 
 Here, the flag `--requestWithdraw` is very important part, which tells the script to initiate the the withdraw request.
-The `tokenIndices` of the tokens, which you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[indices]`.
+The `tokenIndices` of the tokens, which you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[indices]` - do not write the brackets into the command.
```

Comment:
A little nitpick, just because we also use the term "brackets" with another unrelated meaning.
```suggestion
The `tokenIndices` of the tokens, which you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[indices]` - do not write the square brackets into the command.
```